libs/game/game.ts

```diff
@@ -1,8 +1,6 @@
 /**
  * Game transitions and dialog
  **/
-//% color=#8854d0 weight=97 icon="\uf111"
-//% groups='["Gameplay", "Prompt"]'
```

Comment:
I would probably look for the contents of ns.ts here rather than in a separate file, but not a big deal either way

Reply:
This seems to be a common pattern across many common packages. I think it's a good one, helps to remove the guesswork to find the config.